daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala

```diff
@@ -24,7 +24,7 @@ object LanguageVersion {
 
   private[lf] val All = Major.V1.supportedMinorVersions.map(LanguageVersion(Major.V1, _))
 
-  private[lf] val List(v1_6, v1_7, v1_8, v1_11, v1_12, v1_dev) = All
+  val List(v1_6, v1_7, v1_8, v1_11, v1_12, v1_dev) = All
```

Comment:
@remyhaemmerle-da I know you’re not a fan of making things public so if you prefer, I can revert this and duplicate the logic in the tests. That said, I think it is pretty reasonable in this case.